packages/test/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@truffle/test",
+  "description": "Package for testing Solidity contracts.",
+  "license": "MIT",
+  "author": "Tyler Feickert <tyler.feickert@consensys.net>",
+  "homepage": "https://github.com/trufflesuite/truffle/tree/master/packages/test#readme",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/trufflesuite/truffle.git",
+    "directory": "packages/test"
+  },
+  "bugs": {
+    "url": "https://github.com/trufflesuite/truffle/issues"
+  },
+  "version": "0.1.0",
+  "main": "dist/src/index.js",
+  "scripts": {
+    "build": "tsc",
+    "prepare": "yarn build"
+  },
+  "dependencies": {
+    "@truffle/codec": "^0.14.5",
+    "@truffle/compile-common": "^0.8.1",
+    "@truffle/compile-solidity": "^6.0.44",
+    "@truffle/config": "^1.3.38",
+    "@truffle/debugger": "^11.0.10",
+    "@truffle/decoder": "^5.2.30",
+    "@truffle/deployer": "^3.2.97",
+    "@truffle/expect": "^0.1.3",
+    "@truffle/interface-adapter": "^0.5.22",
+    "@truffle/migrate": "^3.3.11",
+    "@truffle/resolver": "^9.0.17",
+    "@truffle/workflow-compile": "^4.0.34",
+    "bn.js": "^5.1.3",
+    "chai": "^4.2.0",
+    "colors": "1.4.0",
+    "debug": "^4.3.1",
+    "mocha": "9.2.2",
+    "original-require": "^1.0.1",
+    "web3-utils": "1.7.4"
+  },
+  "devDependencies": {
+    "@types/bn.js": "^5.1.0",
+    "@types/mocha": "^5.2.7",
```

Comment:
If this is the same version it used previously then I would vote we upgrade in a separate PR. If this is behind the rest of Truffle I'm going to assume there is some reason and that it might not be trivial to upgrade. You ok with that?

Reply:
Yep, I'm cool w/ that.